crates/bevy_asset/src/server/mod.rs

```diff
@@ -963,6 +920,13 @@ pub struct MissingAssetLoaderForTypeNameError {
     type_name: String,
 }
 
+/// An error that occurs when an [`AssetLoader`] is not registered for a given [`Asset`] [`TypeId`].
+#[derive(Error, Debug)]
+#[error("no `AssetLoader` found with the ID '{type_id:?}'")]
+pub struct MissingAssetLoaderForTypeIdError {
+    type_id: TypeId,
```

Comment:
```suggestion
    pub type_id: TypeId,
```

Might as well make things a bit more usable.